src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -112,47 +117,54 @@
           [rn/view
            {:style {:padding-horizontal 12
                     :flex-direction     :row}}
-           [avatar-container message-data]
-           [rn/view
-            {:style {:margin-left 8
-                     :flex        1}}
-            [author message-data]
-            (case content-type
+           [avatar-container message-data is-message-reaction-view?]
+           (into
+            (if is-message-reaction-view?
+              [gesture-handler/scroll-view]
+              [rn/view])
+            [{:style (cond-> {:margin-left 8
```

Comment:
Sometimes I find `cond->` unnecessary or difficult to read. In Clojure, map keys are expected to be nil all the time, so if we take advantage of it, sometimes it's simpler to read:

```clojure
{:margin-left 8
 :flex        1
 :max-height (when is-message-reaction-view (* 0.4 height))}
```

Reply:
Done